types/jwt.d.ts

```diff
@@ -121,11 +121,13 @@ declare namespace fastifyJwt {
   export interface SignOptions extends Omit<SignerOptions, "expiresIn" | "notBefore"> {
     expiresIn: number | string;
     notBefore: number | string;
+    key: string | Buffer

```

Comment:
it seems optional to me - doesn't it?

```suggestion
    key?: string | Buffer

```

Reply:
Yes! Thank you for catching this, my mistake 😥

Reply:
No problem 😄